ci-operator/config/quay/quay-tests/quay-quay-tests-new_ui_add_tag__quay-newui.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+build_root:
```

Comment:
@huweihua-redhat 
I checked the quay-test-master file and it is 
```yaml
build_root:
  image_stream_tag:
    name: release
    namespace: openshift
    tag: golang-1.21
```

now the docker building is failing, maybe related to this.